packages/interpreter/src/helpers/index.ts

```diff
@@ -150,6 +150,6 @@ export function getOperationValue(
       return operands.left || operands.right;
 
     default:
-      throw new InvalidStateException(`Unsupported operator: ${operator}`);
+      throw new InvalidStateException(`ye operator se nahi jamta: ${operator}`);
```

Comment:
This change is not mentioned in the PR description. can you please make sure to mention it? Also, there is an issue in the issue list for this item. Please make sure to refer to the issue when raising the PR.